docs/docs/sourcing-from-databases.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Sourcing from Databases
+title: Abastecimiento de la base de datos
 ---
 
-The Gatsby ecosystem includes built-in connectors to several database systems. These include:
+El ecosistema de Gatsby incluye incorporados conectores a varios sistemas de bases de datos. Éstos incluyen:
 
 - [PostgreSQL](/packages/gatsby-source-pg/?=pg)
 - [MySQL](/packages/gatsby-source-mysql/?=mysql)
 - [Amazon Redshift, SQLite3, Oracle and MSSQL](https://github.com/mrfunnyshoes/gatsby-source-sql)
 - [MongoDB](/packages/gatsby-source-mongodb/)
 - [Firebase](/packages/gatsby-source-firebase/)/[Firestore](/packages/gatsby-source-firestore/)
 
-These plugins will ingest data from your DB and make it available for your page components to query.
+Estos complementos ingerir datos de su DB y los pondrán a disposición para que sus componentes de tu página query.
```

Comment:
I don't consider this to be 100% accurate. I'd change this to:
"`Estos complementos tomarán los datos de tu base de datos y los pondrán a disposición de los componentes de tu página para que sean consultados`".